gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -61,7 +65,9 @@
             <property name="allowByTailComment" value="true"/>
             <property name="allowNonPrintableEscapes" value="true"/>
         </module>
-        <module name="AvoidNestedBlocks"/> <!-- Java Coding Guidelines: Avoid nested blocks -->
+        <module name="AvoidNestedBlocks"> <!-- Java Coding Guidelines: Avoid nested blocks -->
```

Comment:
I don't think I can do that in this PR as they're not on GitHub?  It seems Baseline is somewhat scattered across Stash, GitHub and GHE